pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -279,7 +289,21 @@ func (actuator *OvirtActuator) patchMachine(machine *machinev1.Machine, instance
 	return nil
 }
 
-func (actuator *OvirtActuator) reconcileNetwork(machine *machinev1.Machine, instance *clients.Instance) error {
+	func (actuator *OvirtActuator) getClusterAddress(ctx context.Context) (map[string]int,error){
```

Comment:
Not sure if the tab here is needed or intentional

Reply:
fixed